### DISCUSSION TITLE
any example in react js or ts to connectwalletv2 and contract interaction

### DISCUSSION THREAD

**Discussion by Sriharsh1103** [WalletConnect/walletconnect-monorepo#2904] [Developer Support]
i need a demo on in react connectwalletv2 web3 modal functional calling with contract and how can i customize web3 modal according to our project need ? if any plz share link and or code soo we could implement it

**Comment by ignaciosantise** [comment]
Duplicated of https://github.com/orgs/WalletConnect/discussions/2904
